operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/dependent/workflow/Workflow.java

```diff
@@ -8,18 +8,31 @@
 import java.util.concurrent.Executors;
 
 import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.javaoperatorsdk.operator.api.config.ConfigurationServiceProvider;
 import io.javaoperatorsdk.operator.api.reconciler.Context;
 
+/**
+ * Dependents definition: so if B depends on A, the B is dependent of A.
```

Comment:
I think this is confusing since it's the reverse of what we use for `DependentResource`: the primary resource depends on the dependents…

Reply:
Yep, I think the naming needs to be improved here in general. So this is a directed graph at the end, if reverse direction we could use simply `parents`. Not sure if `dependents` is actually bad, but yes can be confused with dependent resource from other context, maybe just use `child`?